### DISCUSSION TITLE
ProxyJump

### DISCUSSION THREAD

**Discussion by 3f6a** [isontheline/pro.webssh.net#1100] [Q&A]
I need to access a server that needs to go through an entry point bastion server (as described here: https://www.redhat.com/sysadmin/ssh-proxy-bastion-proxyjump).

For this I'd normally use the `ProxyJump` setting of `ssh`. How do I get this behavior with webssh ?

**Comment by isontheline** [comment]
Hello 👋

1. Create a dynamic forward inside WebSSH (which points to your bastion)
2. Launch any SSH connection inside WebSSH

https://webssh.net/documentation/help/networking/dynamic-port-forwarding/

DPF will act same as having a ProxyJump

**Comment by 3f6a** [comment]
Ah I was just reading that. Thanks for the quick reply.

One question: I would like to use the bastion to access some specific servers, while for other servers I would access them directly. Is it possible to tunnel only specific connections through the bastion?

**Reply by isontheline** [reply]
Unfortunately no 😓 I need to work on Jump issue before : https://github.com/isontheline/pro.webssh.net/issues/577

I can't tell you any ETA however
Is a such feature a needed one for you workflow?

**Reply by 3f6a** [reply]
> Is a such feature a needed one for you workflow?

Unfortunately yes. I use these servers for work, some of them directly, and others behind a bastion.

**Reply by isontheline** [reply]
I will try to make my best efforts

**Reply by mscuttari** [reply]
Hi @isonthelin. Thanks for your work, I've been very much enjoying the app in the last months. Do you by any chance have any update regarding the support for ProxyJump? I'm very interested in this feature too. Thanks

**Reply by isontheline** [reply]
Hi @mscuttari 👋

Thank you so much for considering WebSSH 🙏

Yes this feature will be implemented.
Can't give any ETA but hope this summer.

Doesn't the [DPF](https://webssh.net/documentation/help/networking/dynamic-port-forwarding/) could be useful while waiting for Proxy Jump?

**Reply by mscuttari** [reply]
Yes, DPF works fine. It's just a bit annoying having to manually switch between tunnels when logging into servers behind different bastions. Looking forward to your work, thanks again!

**Reply by isontheline** [reply]
@mscuttari I'm upvoting this feature in order to drive the near roadmap update, hope to release it soon
Thanks for your feedback!

**Reply by gliatti** [reply]
> @mscuttari I'm upvoting this feature in order to drive the near roadmap update, hope to release it soon Thanks for your feedback!

This is the only thing keeping me buying the pro version

if you have any workaround, please tell me

**Reply by isontheline** [reply]
@robinportigliatti Only Dynamic Port Forwarding DPF available yet

**Reply by gliatti** [reply]
Too bad, I couldn't make DPF works. I will be waiting then :D
